modules/osdk-scorecard-config.adoc

```diff
@@ -0,0 +1,182 @@
+
+Command arguments -> configuration file -> default
+
+The configuration file must be in YAML format. As the configuration file may be
+extended to allow configuration of all `operator-sdk` subcommands in the future,
+the scorecard's configuration must be under a `scorecard` subsection.
+
+[NOTE]
+====
+Configuration file support is provided by the `viper` package. For more info
+on how viper configuration works, see the `viper` package
+link:https://github.com/spf13/viper/blob/master/README.md[README].
+====
+
+[id="osdk-scorecard-config-args_{context}"]
+== Command arguments
+
+While most of the scorecard tool's configuration is done using a configuration
+configuration file, the following arguments can also be used:
+
+.Scorecard tool arguments
+|===
+|Flag |Type |Description
+
+|`--bundle`, `-b`
+|string
+|The path to a bundle directory used for the bundle validation test.
+
+|`--config`
+|string
+|Path to the scorecard configuration file. The default is
+`<project_dir>/.osdk-scorecard`. File type and extension must be `.yaml`. If a
+configuration file is not provided or found at the default location, the
+scorecard exits with an error.
+
+|`--output`, `-o`
+|string
+|Output format. Valid options are `text` and `json`. The default format is
+`text`, which is designed to be a simpler human readable format. The `json`
+format uses the JSON schema output format used for plug-ins defined later.
+
+|`--kubeconfig`, `-o`
+|string
+|Path to the `kubeconfig` file. It sets the `kubeconfig` internally for internal
+plug-ins.
+
+|`--version`
+|string
+|The version of scorecard to run. The default and only valid option is is
+`v1alpha2`.
+
+|`--selector`, `-l`
+|string
+|The label selector to filter tests on.
+
+|`--list`, `-L`
+|bool
+|If true, only print the test names that would be run based on selector filtering.
+|===
+
+[id="osdk-scorecard-config-file-options_{context}"]
+== Configuration file options
+
+The scorecard configuration file provides the following options:
+
+.Scorecard configuration file options
+|===
+|Option |Type |Description
+
+|`bundle`
+|string
+|Equivalent of the `--bundle` flag. OLM bundle directory path, when specified,
+runs bundle validation.
+
+|`output`
+|string
+|Equivalent of the `--output` flag. If this option is defined by both the
+configuration file and the flag, the flag's value takes priority.
+
+|`kubeconfig`
+|string
+|Equivalent of the `--kubeconfig` flag. If this option is defined by both the
+configuration file and the flag, the flag's value takes priority.
+
+|`plugins`
+|array
+|An array of plug-in names.
+|===
+
+[id="osdk-scorecard-config-plugins_{context}"]
+=== Basic and OLM plug-ins
+
+The scorecard supports the internal `basic` and `olm` plug-ins, which are
+configured by a `plugins` section in the configuration file.
+
+.Plug-in options
+|===
+|Option |Type |Description
+
+|`cr-manifest`
+|[]string
+|Path(s) for CRs being tested. Required if `olm-deployed` is unset or `false`.
```

Comment:
For all of the descriptions, suggest starting with "The". Not critical, but just reads better in my opinion.